src/pydolphinscheduler/core/task.py

```diff
@@ -251,24 +246,17 @@ def timeout_flag(self) -> str:
         """Whether the timeout attribute is being set or not."""
         return TaskTimeoutFlag.ON if self._timeout else TaskTimeoutFlag.OFF
 
-    @property
-    def is_cache(self) -> str:
-        """Whether the cache is being set or not."""
-        if isinstance(self._is_cache, bool):
-            return IsCache.YES if self._is_cache else IsCache.NO
-        else:
-            raise PyDSParamException("is_cache must be a bool")
-
     @property
     def resource_list(self) -> List[Dict[str, Resource]]:
         """Get task define attribute `resource_list`."""
         resources = set()
         for res in self._resource_list:
+
             if type(res) == str:
                 resources.add(
                     Resource(
                         name=res, user_name=self.user_name
-                    ).get_fullname_from_database()
+                    ).get_id_from_database()
```

Comment:
hi @Treasure-u in the latest dev code, we use resource file full path as identify instead of resource id, so in python-sdk python code we will also use full path.

Reply:
thanks for your reply, im sorry it's too late to reply you,i know it and with pydolphin can do better and better